chitchat/src/state.rs

```diff
@@ -303,22 +317,40 @@ impl ClusterState {
 
     pub(crate) fn apply_delta(&mut self, delta: Delta) {
         // Remove nodes to reset.
+        if delta.nodes_to_reset.len() > 0 {
+            tracing::error!(nodes_to_reset=?delta.nodes_to_reset, "nodes to reset");
```

Comment:
Is having to reset nodes an error? info or warning would be better

Reply:
yes that was a mistake. I made it an info.